docs/reference/operators/mobile_package_op_type_support_1.15.md

```diff
@@ -0,0 +1,139 @@
+---
+title: ORT 1.15 Mobile Package Operators
```

Comment:
ops haven't changed since 1.14 so the 2 new files are just a copy of the old info with an updated version number